magento/magento2
# [MFTF] Refactoring of AdminAvailabilityCreditMemoWithNoPaymentTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Go to Admin
2. Start creating a new order using the Create New Customer functionality
3. Add a product to the Order
4. Adjust product price to 0 (via the Custom Price)
5. Place the Order
6. Invoice the Order
7. Check if it is possible to create the Credit Memo



### Resolved issues:
1. [x] resolves magento/magento2#30829: WIP: [MFTF] Refactoring of AdminAvailabilityCreditMemoWithNoPaymentTest
